src/examples/zkapps/voting/test.ts

```diff
@@ -1,17 +1,36 @@
-import { Experimental, Mina, AccountUpdate } from 'snarkyjs';
+import {
```

Comment:
Should this test script be run as part of `./run-integration-tests.sh`, so that voting tested in CI?
(And does this mean the `voting/run.ts` script doesn't need to run in CI anymore? Sorry I haven't really looked at either of these yet)

Reply:
`run.ts` sets the preconditions and parameters for the tests we want to run and then calls into `test.ts` to execute all the tests, so it's being run although maybe a bit confusing :D